lib/phonegap-build/create/zip.js

```diff
@@ -116,7 +116,7 @@ module.exports = {
         // remove zip directory if empty
         exists = fs.existsSync(basepath);
         if (exists) {
-            fs.rmdir(basepath);
+            fs.rmdir(basepath, function () {});
```

Comment:
Since the sync function is used  above in `fs.existsSync`, you might want to use `fs.rmdirSync` instead.

Reply:
Yeah good point!